src/connector/src/sink/encoder/avro.rs

```diff
@@ -59,12 +81,19 @@ impl AvroEncoder {
             schema,
             col_indices,
             avro_schema,
+            header,
         })
     }
 }
 
+pub struct AvroEncoded {
```

Comment:
@liurenjie1024 please check if we use schema registry correctly?

Reply:
LGTM